navigator/backend/src/main/scala/com/digitalasset/navigator/store/platform/PlatformSubscriber.scala

```diff
@@ -70,6 +70,7 @@ class PlatformSubscriber(
   // ----------------------------------------------------------------------------------------------
   // Lifecycle
   // ----------------------------------------------------------------------------------------------
+  @scala.annotation.nowarn("msg=discarded non-Unit value")
```

Comment:
We shouldn't abuse warning suppression. Rather, either:
- explicitly drop the non-`Unit` value by binding it to `_` before returning (i.e. `val _ = ...`), or
- decorate the expression rather than then method and possibly see if there's a category that fits rather then matching the message (i.e. see if `expression: @annotation.nowarn("cat=w-flag-value-discard")` works)

Further note: the `scala` package is always in scope, no need to put it in the fully qualified name.